cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -179,6 +188,35 @@ func (a *StaticAutoscaler) cleanUpIfRequired() {
 	a.initialized = true
 }
 
+func (a *StaticAutoscaler) propagateClusterSnapshot(nodes []*apiv1.Node, scheduledPods []*apiv1.Pod) errors.AutoscalerError {
```

Comment:
You're not really propagating anything here; updateClusterSnapshot?

Reply:
Renamed to `initializeClusterSnapshot`